main.go

```diff
@@ -24,5 +26,21 @@ func main() {
 		klog.Fatal(configError)
 	}
 
+	// Load PerformanceProfile in Kruize Instance first
+	perfProfileInitialized := false
+	for !perfProfileInitialized {
+		klog.Info("Initializing performanceProfile.")
+		perfProfileInitialized = kruize.InitPerformanceProfile()
+		if perfProfileInitialized {
+			klog.Info("Initialized performanceProfile.")
+			break
```

Comment:
We need to check periodically if this is still true., in case the kruize container dies in between.

Reply:
@SherinV Dev Preview : We will check during getPerformanceProfile 
                   Tech Preview : We need a initContainer for Kruize , which will ensure the presence of PerformanceProfile